packages/elements/src/icon/index.ts

```diff
@@ -230,10 +230,15 @@ export class Icon extends BasicElement {
    * @returns {void}
    */
   private setPrefix(): void {
+    /**
+     * This prefix for individual icons allows supporting custom prefix of self-managed icons.
+     * One of the use case is building EFX components.
```

Comment:
I think the second sentence is not necessary. Also Should we mention that cdn-prefix is high priority than cdn-sprite?

Reply:
fixed!